Formula/mold.rb

```diff
@@ -1,8 +1,8 @@
 class Mold < Formula
   desc "Modern Linker"
   homepage "https://github.com/rui314/mold"
-  url "https://github.com/rui314/mold/archive/v1.1.1.tar.gz"
-  sha256 "47c5ddfe60beffc01da954191c819d78924e4d1eb96aeebfa24e1862cb3a33f9"
+  url "https://github.com/rui314/mold/archive/v1.2.tar.gz"
```

Comment:
should this be `1.2.0` instead of `1.2`?

Reply:
You’d think so but at the time I used `brew bump-formula-pr` the `1.2.0` tag didn’t even exist.  🤷

Reply:
I’ll change it over now that it does.

Reply:
```suggestion
  url "https://github.com/rui314/mold/archive/v1.2.0.tar.gz"
```